src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -57,6 +57,22 @@ public override Conversion GetMethodGroupDelegateConversion(BoundMethodGroup sou
 
             Debug.Assert(methodSymbol == ((NamedTypeSymbol)destination).DelegateInvokeMethod);
 
+            // If synthesizing a delegate with `params` array, check that `ParamArrayAttribute` is available.
+            if (methodSymbol is SynthesizedDelegateInvokeMethod)
+            {
+                foreach (var parameter in methodSymbol.Parameters)
```

Comment:
>foreach (var parameter in methodSymbol.Parameters) [](http://example.com/codeflow?start=16&length=50)

It feels like we need to check only the last parameter.

Reply:
Thanks, you're right, changed.